druid-shell/src/windows/mod.rs

```diff
@@ -1191,6 +1232,22 @@ impl<'a> WinCtx<'a> for WinCtxImpl<'a> {
             SetCursor(cursor);
         }
     }
+
+    /// Request a timer event.
+    ///
+    /// The return value is an identifier.
+    fn request_timer(&mut self, deadline: std::time::Instant) -> usize {
```

Comment:
okay taking a closer look now, this feels weird; we're basically using `0` as a null return value, but are we expecting the user to handle this?

'no' is okay, but i think it should be better documented. Otherwise this should probably be returning an `Option`, which also feels bad. This is probably okay, but it feels a bit iffy; we're designing a general API around a platform convention.

Reply:
I don't think it'll fail, or if it does it's proabably ok to panic. (there might be a limit but it's extremely unlikely to hit it in practice). I'm basically just avoiding 0 in the call to SetTimer because it has a special purpose.